astro/src/components/bikes/BikesSearchList.astro

```diff
@@ -0,0 +1,33 @@
+---
+import type { BikesSearchCardProps } from "@components/cards/BikesSearchCard.astro";
+import BikesSearchCard from "@components/cards/BikesSearchCard.astro";
+
+export interface BikesSearchListProps {
+  items: BikesSearchCardProps[];
+  class?: string;
+}
+
+const { items = [], class: classes } = Astro.props as BikesSearchListProps;
+---
+
+{
+  items?.length > 0 ? (
+    <ul
+      class:list={[
+        "my-4 md:my-10 grid gap-4 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4",
+        {},
+        classes
+      ]}
+    >
+      {items.map((item) => (
+        <BikesSearchCard {...item} />
+      ))}
+    </ul>
+  ) : (
+    <>
+      <p class="text-center my-4 md:my-10">
```

Comment:
this little snippet appears in the previous component. maybe it would be a good practice to make it a separated one (in order not to repeat code) and also, again, use i18n